pkgs/os-specific/linux/zfs/stable.nix

```diff
@@ -0,0 +1,26 @@
+{ callPackage
+, kernel ? null
+, stdenv
+, linuxKernel
+, ...
+} @ args:
+
+let
+  stdenv' = if kernel == null then stdenv else kernel.stdenv;
+in
+callPackage ./generic.nix args {
+  # check the release notes for compatible kernels
+  kernelCompatible =
+    if stdenv'.isx86_64
+    then kernel.kernelOlder "6.3"
+    else kernel.kernelOlder "6.2";
+  latestCompatibleLinuxPackages =
+    if stdenv'.isx86_64
+    then linuxKernel.packages.linux_6_1
+    else linuxKernel.packages.linux_6_2;
```

Comment:
Did you intentionally flip the then and else? I'm currently using zfs with kernel 6.2 and this would mean a downgrade to 6.1 for me.